### ISSUE TITLE
Your documentation needs to be SIMPLE, APPSettings for dotnet 6.0

### ISSUE THREAD

**Issue by pthorsley** [dotnet/AspNetCore.Docs#25565]
Rick,

Your documentation here is terrible and very ambiguous....
You should focus on the term "AppSettings" since that is what we all google for!!
Next you need to show comparison between dotnet core 2.1, 3.1, and 6.0 since you do it different in each one..... 
MEGA CONFUSING , for something so simple....

HERE IS THE FIX for dotnet c# 6.0:
```

builder.Configuration.SetBasePath(builder.Environment.ContentRootPath)
    .AddJsonFile("appsettings.json", optional: true, reloadOnChange: true)
    .AddJsonFile($"appsettings.{builder.Environment.EnvironmentName}.json", optional: true);

builder.Configuration.AddEnvironmentVariables();

builder.Services.Configure<AppSettings>(builder.Configuration);
```

Thanks

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 0eb4d762-3910-2c57-8741-3306c15537b6
* Version Independent ID: ba2254a0-ea8e-9b28-e94b-edd0fc58c043
* Content: [Configuration in ASP.NET Core](https://docs.microsoft.com/en-us/aspnet/core/fundamentals/configuration/?view=aspnetcore-6.0)
* Content Source: [aspnetcore/fundamentals/configuration/index.md](https://github.com/dotnet/AspNetCore.Docs/blob/main/aspnetcore/fundamentals/configuration/index.md)
* Product: **aspnet-core**
* Technology: **aspnetcore-fundamentals**
* GitHub Login: @Rick-Anderson
* Microsoft Alias: **riande**

**Comment by Rick-Anderson** [comment]
> Next you need to show comparison between dotnet core 2.1, 3.1, and 6.0 since you do it different in each one.....
We don't show difference between previous versions in the primary doc. Differences are shown in the migration docs. Use the version selector if you're not on the current released version.

> HERE IS THE FIX for dotnet c# 6.0:

Fix for what?

**Comment by pthorsley** [comment]
fix for what? bad docs is what

another reason we are now moving to golang and react...
